docs/reference/search/search-your-data/semantic-search-deploy-model.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[semantic-search-deployed-nlp-model]]
+=== Tutorial: semantic search with a deployed model
+
+++++
+<titleabbrev>Semantic search with deployed model</titleabbrev>
+++++
+
+[IMPORTANT]
+====
+* For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.
+* This tutorial was written before the {infer} endpoint and `semantic_text` options were introduced. Today we have simpler options for performing semantic search. 
```

Comment:
```suggestion
* This tutorial was written before the {infer} endpoint and `semantic_text` field type was introduced. Today we have simpler options for performing semantic search. 
```

Also can we add links to the [inference API](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) and [semantic_text field type](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-text.html) documentation here?